packages/vuetify/src/directives/colorable/__tests__/colorable.spec.ts

```diff
@@ -0,0 +1,241 @@
+      expect(wrapper.element.style.color).toEqual('rgba(0, 1, 2, 0.5)')
+      expect(wrapper.element.style.caretColor).toEqual('rgba(0, 1, 2, 0.5)')
+    })
+
+    it('should set element styles with theme color', async () => {
+      const wrapper = mountFunction()
+
+      wrapper.setData({ color: 'red' })
+      expect(wrapper.element.style.color).toEqual('rgb(244, 67, 54)')
+      expect(wrapper.element.style.caretColor).toEqual('#f44336')
+
+      wrapper.setData({ color: 'red lighten-1' })
+      expect(wrapper.element.style.color).toEqual('rgb(239, 83, 80)')
+      expect(wrapper.element.style.caretColor).toEqual('#ef5350')
+
+      wrapper.setData({ color: 'primary' })
+      expect(wrapper.element.style.color).toEqual('rgb(25, 118, 210)')
+      expect(wrapper.element.style.caretColor).toEqual('#1976d2')
+    })
+  })
+
+  describe('BgColor', () => {
+    beforeEach(() => {
+      mountFunction = () => {
+        return mount(Vue.component('test', {
+          directives: { ...Colorable },
+          data: () => ({
+            color: '',
+          }),
+          render (h) {
+            const data = {
+              directives: [{
+                name: 'bgColor',
+                value: this.color,
+              }],
+            }
+            return h('div', data)
+          },
+        }), {
+          mocks: {
+            $vuetify: {
+              theme: {
+                currentTheme: {
+                  primary: '#1976d2',
+                },
+              },
+            },
+          },
+        })
+      }
+    })
+
+    it('should update element styles reactively', async () => {
+      const wrapper = mountFunction()
+
+      wrapper.setData({ color: '#01f' })
+      expect(wrapper.element.style.backgroundColor).toEqual('rgb(0, 17, 255)')
+      expect(wrapper.element.style.borderColor).toEqual('#01f')
+
+      wrapper.setData({ color: 'rgb(255, 255, 0)' })
+      expect(wrapper.element.style.backgroundColor).toEqual('rgb(255, 255, 0)')
+      expect(wrapper.element.style.borderColor).toEqual('rgb(255, 255, 0)')
+    })
+
+    it('should set element styles with theme color', async () => {
+      const wrapper = mountFunction()
+
+      wrapper.setData({ color: 'red' })
+      expect(wrapper.element.style.backgroundColor).toEqual('rgb(244, 67, 54)')
+      expect(wrapper.element.style.borderColor).toEqual('#f44336')
+
+      wrapper.setData({ color: 'red lighten-1' })
+      expect(wrapper.element.style.backgroundColor).toEqual('rgb(239, 83, 80)')
+      expect(wrapper.element.style.borderColor).toEqual('#ef5350')
+
+      wrapper.setData({ color: 'primary' })
+      expect(wrapper.element.style.backgroundColor).toEqual('rgb(25, 118, 210)')
+      expect(wrapper.element.style.borderColor).toEqual('#1976d2')
+    })
+  })
+
+  describe('BorderColor', () => {
+    beforeEach(() => {
+      mountFunction = (modifiers?: BorderModifiers) => {
+        return mount(Vue.component('test', {
```

Comment:
This is also used here: https://github.com/vuetifyjs/vuetify/pull/10866/files#diff-b98e7199f4155616733cb13749c6e817R26

An example of what we have done with other components: https://github.com/vuetifyjs/vuetify/blob/85f963eea6f872a67761534737158519f0446df2/packages/vuetify/src/components/VItemGroup/__tests__/VItemGroup.spec.ts#L19